### ISSUE TITLE
Custom user-agent

### ISSUE THREAD

**Issue by hochraldo** [openSUSE/zypper#382]
Is it possible to set a custom user agent when zypper connects to repositories. The problem is that in some corporate environment the proxy only allows specific user-agent strings. Would be nice if such a feature does not yet exist if it could be added (as it possible to do with apt)

**Comment by mlandres** [comment]
@hochraldo If an option in `/etc/zypp.conf` (e.g. `download.user_agent`) suits your needs, it can be added quite easily. This would then affect all downloads done by zypper and in YAST,PackageKit at least the repo and package management related ones.

**Comment by hochraldo** [comment]
Yes, that would totally fit my need. In general there seems to be a need for this (not only for my, our company), as a reference also how apt supports this: https://manpages.debian.org/unstable/apt/apt-transport-http.1.en.html as well.
